CONTRIBUTING.md

```diff
@@ -139,7 +146,7 @@ go build -o pinniped ./cmd/pinniped
 1. Run the Pinniped integration tests:
 
    ```bash
-   source /tmp/integration-test-env && go test -v -count 1 -timeout 0 ./test/integration
+   ulimit -n 512 && source /tmp/integration-test-env && go test -v -count 1 -timeout 0 ./test/integration
```

Comment:
I wonder if it would be helpful to also have an example and some instructions on how to run a single integration test. 
e.g. to test the impersonation proxy, run 
`source /tmp/integration-test-env && go test -v -count 1 -timeout 0 ./test/integration -run TestIntegrationParallel/TestImpersonationProxy`

The parallel/serial stuff may not be obvious to someone new to the project.

Reply:
Great idea. Added!